drivers/i2c/Kconfig

```diff
@@ -54,3 +50,7 @@ config I2CMULTIPLEXER_PCA9540BDP
 endmenu # I2C Multiplexer Support
 
 endif
```

Comment:
@patacongo This is the `endif` for the corresponding `if I2c`

`config ARCH_HAVE_I2CRESET` comes after it. Therefore, `ARCH_HAVE_I2CRESET` does NOT DEPEND on I2C and is unconditionally declared. This is correct.